lib/vrl/tests/tests/internal/closure_arg_kind.vrl

```diff
@@ -0,0 +1,38 @@
+#               "timestamp": "2020-09-14T19:09:29.039Z"
+#             },
+#             "log_group": "/test",
+#             "log_stream": "test",
+#             "message_type": "DATA_MESSAGE",
+#             "owner": "111111111111",
+#             "subscription_filters": [
+#               "Destination"
+#             ]
+#           },
+#           {
+#             "log_events": {
+#               "id": "35683658089659183914001456229543810359430816722590236673",
+#               "message": "{\"bytes\":17707,\"datetime\":\"14/Sep/2020:11:45:41 -0400\",\"host\":\"109.81.244.252\",\"method\":\"GET\",\"protocol\":\"HTTP/2.0\",\"referer\":\"http://www.investormission-critical.io/24/7/vortals\",\"request\":\"/scale/functionalities/optimize\",\"source_type\":\"stdin\",\"status\":502,\"user-identifier\":\"feeney1708\"}",
+#               "timestamp": "2020-09-14T19:09:29.041Z"
+#             },
+#             "log_group": "/test",
+#             "log_stream": "test",
+#             "message_type": "DATA_MESSAGE",
+#             "owner": "111111111111",
+#             "subscription_filters": [
+#               "Destination"
+#             ]
+#           }
+#         ]
+
+
+
+. = {"message":"{\n  \"messageType\": \"DATA_MESSAGE\",\n  \"owner\": \"111111111111\",\n  \"logGroup\": \"/test\",\n  \"logStream\": \"test\",\n  \"subscriptionFilters\": [\n    \"Destination\"\n  ],\n  \"logEvents\": [\n    {\n      \"id\": \"35683658089614582423604394983260738922885519999578275840\",\n      \"timestamp\": 1600110569039,\n      \"message\": \"{\\\"bytes\\\":26780,\\\"datetime\\\":\\\"14/Sep/2020:11:45:41 -0400\\\",\\\"host\\\":\\\"157.130.216.193\\\",\\\"method\\\":\\\"PUT\\\",\\\"protocol\\\":\\\"HTTP/1.0\\\",\\\"referer\\\":\\\"https://www.principalcross-platform.io/markets/ubiquitous\\\",\\\"request\\\":\\\"/expedite/convergence\\\",\\\"source_type\\\":\\\"stdin\\\",\\\"status\\\":301,\\\"user-identifier\\\":\\\"-\\\"}\"\n    },\n    {\n      \"id\": \"35683658089659183914001456229543810359430816722590236673\",\n      \"timestamp\": 1600110569041,\n      \"message\": \"{\\\"bytes\\\":17707,\\\"datetime\\\":\\\"14/Sep/2020:11:45:41 -0400\\\",\\\"host\\\":\\\"109.81.244.252\\\",\\\"method\\\":\\\"GET\\\",\\\"protocol\\\":\\\"HTTP/2.0\\\",\\\"referer\\\":\\\"http://www.investormission-critical.io/24/7/vortals\\\",\\\"request\\\":\\\"/scale/functionalities/optimize\\\",\\\"source_type\\\":\\\"stdin\\\",\\\"status\\\":502,\\\"user-identifier\\\":\\\"feeney1708\\\"}\"\n    }\n  ]\n}\n","request_id":"ed1d787c-b9e2-4631-92dc-8e7c9d26d804","source_arn":"arn:aws:firehose:us-east-1:111111111111:deliverystream/test","timestamp":"2020-09-14T19:12:40.138Z"}
+parsed = parse_aws_cloudwatch_log_subscription_message!(.message)
+. = unnest(parsed.log_events)
+map_values(.) -> |value| { events = del(value.log_events) ; value |= events }
```

Comment:
```suggestion
map_values(.) -> |value| { event = del(value.log_events) ; value |= event }
```
I also guess, for readability, this should now be a single "event" so dropping the plural..